core/primitives/src/stateless_validation.rs

```diff
@@ -24,16 +25,31 @@ type SignatureDifferentiator = String;
 #[derive(Debug, Clone, PartialEq, Eq, BorshSerialize, BorshDeserialize)]
 pub struct EncodedChunkStateWitness(Box<[u8]>);
 
+pub type ChunkStateWitnessSize = usize;
```

Comment:
Our past use of plain type aliases has proven to result in a not-great experience: as the codebase expanded, there have been cases of type mixups that had led to bugs.

On the other hand, this is currently used as a return type only where an `usize` would otherwise be perfectly adequate, so I think this is probably alright to leave as-is.